product-manual/koala-plays.mdx

```diff
@@ -0,0 +1,116 @@
+
+You can choose to set up auto-assignment with lead routing rules to dole out leads to the rest of your team, or you can leave leads unassigned. If they are unassigned, they’ll only show up in the Play (i.e. you’ll have to navigate to the Play to see them, assign them, or work them).
+
+If you have assignment happening, reps will have 3 primary ways they can interact with the leads.
+
+1. **In the app**: They will see a new Inbox item in their navigation menu, with a little red dot if they have any leads in their inbox to work.
+2. **In their email**: If notifications are turned on they can receive emails of new assigned leads.
+3. **In Slack**: If notifications are turned on they can receive Slack DMs about new assigned leads, and can join a shared channel for new leads as they get added to the play.
+
+You can also enable notifications so you and your team know when there are new leads. We have two types of notifications you can configure in a couple clicks, today:
+
+1. Enrollment notifications
+2. Assignment notifications
+
+### Enrollment notifications
+
+You can subscribe to a Slack channel to see all new leads in the play that havent already been ack’d or completed or disqualified.
+
+If you enable @ mentions, users who are assigned the lead will be invited to the channel (once only) with their name mentioned on the corresponding lead.
+
+Koala will send alerts as new leads are added to the play with the most relevant data, AI research, and key insights automatically included in the Slack message. 
+
+You can take further action directly from Slack to see more insights, suggested messaging, as well as Claim, Assign, or Dismiss the lead, without having to leave Slack.
+
+### Assignment notifications
+
+You can subscribe to both *email and Slack DMs* for this notification. Whenever leads are assigned to you (either manually, or automatically) you’ll receive a DM in Slack or an email digest of new assigned leads. 
+
+Note: This is configured at the play level for all assignees, and cannot currently be configured per user.
+
+## How do I take action on leads?
+
+There are three places you can take action:
+
+1. From the Play dashboard – you can see the full table view of all the leads or accounts added to the play and you can assign, dismiss, email or add to sequence.
+2. From the Inbox – reps will get an inbox containing only the stuff assigned to them. Admins can see an Inbox Dashboard that shows you how leads are being enrolled, worked, or dismissed by user or by play, with some high level stats about daily/weekly trends.
+3. From Slack – you can subscribe to notifications and do things like AI Research, assignment, and dismissing leads directly from Slack. However, actions like emailing or adding to a sequence still require navigating into the app first.
+
+## Inbox
```

Comment:
Inbox rep experience

Reply:
Ah I think this is good from a leadership perspective but we probably need a separate section/page for Inbox